optional slot mappings (#7171)
RasaHQ/rasa: docs/docs/default-actions.mdx

This action undoes the last user-bot interaction. It can be triggered by the user
by sending a "/back" message to the assistant if the [RulePolicy](./policies.mdx#rule-policy) is configured.
                                                                                                                                                                                                                                                                        |
## Form Action

> By default Rasa Open Source uses the inbuilt `FormAction` for processing any
> [form logic](forms.mdx). You can override this default action with a custom action by
> adding a custom action with the form's name to the domain. This can e.g. be used to use
> the deprecated `FormAction` which is part of the Rasa SDK and used to implement forms
> for Rasa Open Source 1.

Reviewer: ```suggestion
By default Rasa Open Source uses `FormAction` for processing any
[form logic](forms.mdx). You can override this default action with a custom action by
adding a custom action with the form's name to the domain. This can e.g. be used to use
the deprecated `FormAction` which is part of the Rasa SDK and used to implement forms
for Rasa Open Source 1.
```

Reviewer 2: @wochinge didn't I say that we should just remove the "caution" note here and just state in the text that overriding should ONLY be done during migration? So basically say this directly in the text, rather than saying "This can e.g. be used".

Author: Yes, but I haven't worked that in yet.

Author: @melindaloubser1 Merged your suggestion with Akela's

Reviewer 2: haha ok sorry, i assumed since you requested Melinda for review you weren't going to ;)

:::caution

> Overriding the default action for forms should **only** be used during the process of
> migrating from Rasa Open Source 1 to 2. It is highly discouraged to use this to
> implement custom form versions. Instead we recommend to use the customizations which
> are described in the [Advanced Usage](forms.mdx#advanced-usage) section of forms.

Reviewer: ```suggestion
Overriding the default action for forms should **only** be used during the process of
migrating from Rasa Open Source 1 to 2. Do not to use this to
implement custom form versions. To customize forms, see the [Advanced Usage](forms.mdx#advanced-usage) section of forms.
```
May be overkill on the simplification but "it's recommended/discouraged" is  🙅‍♀️

Author: Deleted the caution section as Akela suggested. Merged some parts of your suggestion to the new version
